src/tools/vmap4_extractor/wdtfile.cpp

```diff
@@ -32,40 +33,45 @@ char * wdtGetPlainName(char * FileName)
 
 extern CASC::StorageHandle CascStorage;
 
-WDTFile::WDTFile(char* file_name, char* file_name1):WDT(CascStorage, file_name), gnWMO(0)
+WDTFile::WDTFile(char const* storagePath, std::string mapName, bool cache)
+    : _file(CascStorage, storagePath), _mapName(std::move(mapName))
 {
-    filename.append(file_name1,strlen(file_name1));
+    if (cache)
+    {
```

Comment:
Is this due to coding standard or why not use  std::make_unique here and skip the include of common here?

Reply:
including common is no longer bad since cleanup + fixing header dependencies (it no longer does crazy things like pull in mysql dep into extractors)
that and simply consistency - using the same thing everywhere